libs/features/mas/src/hydrate.js

```diff
@@ -186,28 +156,67 @@ function createSpectrumButton(cta, strong, aemFragmentMapping, cardVariant) {
     return spectrumCta;
 }
 
-function processConsonantButton(cta, strong) {
+function processConsonantButton(cta, variant, treatment, aemFragmentMapping) {
     cta.classList.add('con-button');
-    if (strong) {
+    if (treatment === 'outline') {
+        cta.classList.add('outline');
+    }
+    if (variant === 'accent') {
         cta.classList.add('blue');
+    } else if (variant === 'black') {
+        cta.classList.add('fill');
+    }
+    const size = aemFragmentMapping.ctas.size ?? 'm';
+    if (size === 's') {
```

Comment:
nit: this can be written inline
```suggestion
    if (size === 's') cta.classList.add('button-s');
```